UI/UnoGoodReads/UnoGoodReads/UnoGoodReads.Shared/Styles/MainDictionary.xaml

```diff
@@ -1,6 +1,5 @@
 ﻿<ResourceDictionary xmlns:uen="using:Uno.Extensions.Navigation.UI" xmlns:uer="using:Uno.Extensions.Reactive.UI" xmlns:utu="using:Uno.Toolkit.UI" xmlns:muxc="using:Microsoft.UI.Xaml.Controls" xmlns:um="using:Uno.Material" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" xmlns:d="http://schemas.microsoft.com/expression/blend/2008" xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
-  <ResourceDictionary.ThemeDictionaries>
-    <ResourceDictionary x:Key="Light">
+  <ResourceDictionary.ThemeDictionaries>    
```

Comment:
@aesteves900 why did you remove the Light/Dark options in the ThemeDictionaries?

Reply:
Yes, I removed it because there is no switch button.